### ISSUE TITLE
Docs Request: how to put check-hidden into .codespellrc

### ISSUE THREAD

**Issue by jamesbraza** [codespell-project/codespell#2718]
If I have the following `.codespellrc` for `codespell==2.2.2`:

```rc
[codespell]
check-filenames = True
check-hidden = True
```

And I run the below:

```bash
> codespell foo
usage: codespell [-h] [--version] [-d] [-c] [-w] [-D DICTIONARY] [--builtin BUILTIN-LIST] [--ignore-regex IGNORE_REGEX] [-I FILE] [-L WORDS] [--uri-ignore-words-list WORDS] [-r REGEX] [--uri-regex URI_REGEX] [-s] [--count] [-S SKIP]
                 [-x FILE] [-i INTERACTIVE] [-q QUIET_LEVEL] [-e] [-f] [-H] [-A LINES] [-B LINES] [-C LINES] [--config CONFIG] [--toml TOML]
                 [files ...]
codespell: error: unrecognized arguments: True
```

So this seems to be wrong.  From a GitHub advanced search, I see this:

```rc
[codespell]
check-filenames =
check-hidden =
```

However, this is not clear to me, would the checks be enabled or disabled here?

Can you please document how to put `check-filenames` and `check-hidden` into the `README`?  Thank you in advance

**Comment by DimitriPapadopoulos** [comment]
The [README](https://github.com/codespell-project/codespell/blob/master/README.rst) attempts to document that usign examples:
```ìni
    [codespell]
    skip = *.po,*.ts,./src/3rdParty,./src/Test
    count =
    quiet-level = 3
```
`--skip SKIP` → `skip = SKIP`
`--count` → `count =`
`--quiet-level QUIET_LEVEL` → `quiet-level = QUIET_LEVEL`

**Comment by jamesbraza** [comment]
Yeah I did see that in the `README` actually, and understand the mapping between CLI names and config names.

However, it's not clear how to use `check-filenames`/`check-hidden`.  Do we just need their presence, or should we set to `True`?

```none
[codespell]
check-filenames = True
check-filenames = 
```

**Comment by DimitriPapadopoulos** [comment]
`--count` → `count =`

**Comment by jamesbraza** [comment]
Ah I see, that clears me up... and thank you!  Please feel free to close this issue.

---

One docs aspect I like in other Python tools is they have sample complete configs:
- `pylint` documents all at once: https://github.com/PyCQA/pylint/blob/main/examples/pylintrc
- `flake8` and `isort` document on a case-by-case basis:
    - https://flake8.pycqa.org/en/latest/user/options.html
    - https://pycqa.github.io/isort/docs/configuration/options.html

If `codespell` had one of these, I think it'd be helpful

**Comment by DimitriPapadopoulos** [comment]
Yes, there is room for improvement.

In this case, knowing the solution helped me understand the example in the documentation, instead of the other way round.
